content/pages/about/index.md

```diff
@@ -2,17 +2,8 @@
 title: 'About'
 date: 2018-12-06T09:29:16+10:00
 layout: 'aboutlayout'
-heroHeading: 'About Us'
-heroSubHeading: "We've been supporting students since 2010. Join us as we celebrate 10 years of learning!"
+heroHeading: 'Supporting Students Since 2010'
```

Comment:
Can we remove the capitalization here? 

Eg. `Supporting students since 2010`

Small nitpick I know, it's just to maintain consistency across the website. Capitalization should be used for acronyms (eg. Frequently Asked Questions because the acronym is _FAQ_), but not titles.

Reply:
Actually nvm, I see why you capitalized it here. 

It's a bit long, can we use something else? The 10 years is implied by the text right underneath. Perhaps:

- `Supporting Students`
- `Supporting Learners`
- etc...

![image](https://user-images.githubusercontent.com/43160110/84451768-f6a30b00-ac21-11ea-9f20-9b6def358457.png)

Reply:
N.B. It's okay if it doesn't fit into 1 line on mobile, but it would be nice to fit it into one line on desktop.